department-of-veterans-affairs/vets-api
# Update vets-json-schema

## Summary

Updates vets-json-schema to use profileAddress for addresses in Pension. 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/vets-json-schema/pull/911
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86037

## Testing done

- Fill out a pension claim on the `master` branch. Have at least one dependent with a street that is longer than 51 and shorter than 100 characters. You will see no error on the front end, but when you try to submit, it will fail.
- Checkout this branch and run `bundle install`
- Attempt to submit the same claim from before and it will succeed.

## What areas of the site does it impact?
Pensions

## Acceptance criteria
